src/gluonts/model/estimator.py

```diff
@@ -30,14 +32,12 @@ class Estimator:
     __version__: str = gluonts.__version__
 
     prediction_length: int
-    lead_time: int
+    lead_time: int = 0
```

Comment:
Something that should work is to do 
```suggestion
    lead_time: int = field(default=0, metadata={"ge": 0})
```

Then pydantic does the bounds checking for us.